src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-apprunner.mdx

```diff
@@ -0,0 +1,163 @@
+
+## Features
+Collect and send telemetry data to New Relic from your apps and services running on AWS Apprunner containers using our integration. Monitor your services, query incoming data, and build dashboards to observe everything at a glance. 
+
+## Activate integration
+
+To enable this integration follow standard procedures to [Connect AWS services to New Relic](/docs/infrastructure/infrastructure-integrations/getting-started/connect-aws-integrations-infrastructure).
+
+## Configuration and polling 
+
+You can change the polling frequency and filter data using [configuration options](/docs/integrations/new-relic-integrations/getting-started/configure-polling-frequency-data-collection-cloud-integrations).
+
+[Default polling information](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-polling-intervals-infrastructure-integrations) for the AWS integrations:
+- New Relic polling interval: 5 minutes
+- Amazon CloudWatch data interval: 1 minute or 5 minutes
+
+
+## Find and use data
+To find your integration data, go to your account on [one.newrelic.com ](one.newrelic.com).
+
+For logs:
+On the left column select **Logs > Attributes > AWS** and select **logGroup** and/or **logStream**
+
+For metrics:
+On the left column select **Data explorer > Metrics** and filter by `aws.apprunner`
+
+## Metric data
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+      Metric (min, max, average)  
+      </th>
+      <th>
+      Unit  
+      </th>
+      <th>
+      Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+      `2xxStatusResponses`
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Number of successfull requests to the service
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `4xxStatusResponses`  
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Number of failed requests to the service
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `ActiveInstances`
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Number of active instances
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `CPUUtilization`
+      </td>
+      <td>
+      percentage  
+      </td>
+      <td>
+      Instance CPU loading
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `MemoryUtilization`
+      </td>
+      <td>
+      percentage  
+      </td>
+      <td>
+      Instance memory utilization
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `RequestLatency`
+      </td>
+      <td>
+      ms  
+      </td>
+      <td>
+      How long a request takes
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Requests`
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Total number of requests
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+
+## How to use your data
```

Comment:
What other examples should we have here? In the "missing sections" of the g-doc, we have `metrics` `events` and `dashboards`. I worry that metrics and events are commonplace to link here. I'm happy to be convinced otherwise though!